drf_rehive_extras/filter_backend.py

```diff
@@ -0,0 +1,25 @@
+from collections import OrderedDict
+
+from django_filters import rest_framework as filters
+
+
+class RehiveRestFilterBackend(filters.DjangoFilterBackend):
+    def get_ordering(self, request, queryset, view):
```

Comment:
I am not sure this is the correct approach. You are now defining a bunch of ordering options on the filter backend level, rather than on the individual view or filterset_class, which means that you have essentially hardcoded the options to only: `created` or `updated` (ASC or DESC), and defaulting to `-created`.  

What if we want to add an additional `orderby` field in future?  For instance, the `executed` field on transactions or some other timestamp or unique field that may exist on other models. The `drf-rehive-extras` library is meant to be generic and work in any Rehive-like environment, not just the specific list of orderby filters we currently need.

I have two OPTIONS that could work as an alternative:

**Option 1**

I wonder if it is possible to update this method: https://github.com/encode/django-rest-framework/blob/master/rest_framework/pagination.py#L800

So that it tries to get its ordering value from the `filterset_class` first and then if `filterset_class` doesn't exist it tries the `filter_class` as per normal. This way we could perhaps only define the options on the `filterset_class` and can make the CursorPagination be generic and usable without having the allowed orderby fields pre-defined: https://github.com/rehive/drf-rehive-extras/blob/master/drf_rehive_extras/pagination.py#L28

This option may be complex as I am not sure if the `filterset_class` has been initiated at this point. But maybe you can take a look and see if there is a way this could work.

**Option 2**

An alternative way to maybe achieve what we want to achieve on the `CursorPagination` is to modify the `ListModelMixin` to also include the `OrderingFilter` is the `filter_backends` list over here: https://github.com/rehive/drf-rehive-extras/blob/master/drf_rehive_extras/mixins.py#L83 

You can read about the `OrderingFilter` here: https://www.django-rest-framework.org/api-guide/filtering/#orderingfilter

You will have to also modify the `OrderingFilter` (perhaps in this `drf-rehive-extras` library) to specifically only work with the `orderby` GET param instead of the current default of `ordering` as `orderby` is what we currently expect in the API.

And then, we will update the views where ordering is allowed to have a list of fields that are supported, like:  `ordering_fields = ['created', 'updated']`.  For places where custom orderby fields are too complex for cursor pagination (as is the case on the platform sometimes) we can leave those fields on the `filterset_class` and have them only work for `PageNumberPagination` but in general we will be moving away from supporting these complex ordering options anyway.

---

Please chat to me re. this via Slack, and we can try work out the best solution. I think we are slowly figuring out a good approach.